config/authorino/rbac/service_editor_role.yaml

```
@@ -0,0 +1,23 @@
+# permissions for end users to edit services.
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: service-editor-role
```

Perhaps...

```suggestion
  name: authconfig-editor-role
```

WDYT?

---

I changed the name of the yaml file too